src/client-side-metrics/operation-metrics-collector.ts

```
@@ -17,10 +17,23 @@ import {IMetricsHandler} from './metrics-handler';
 import {MethodName, StreamingState} from './client-side-metrics-attributes';
 import {grpc} from 'google-gax';
 import * as gax from 'google-gax';
-const root = gax.protobuf.loadSync(
-  './protos/google/bigtable/v2/response_params.proto'
-);
-const ResponseParams = root.lookupType('ResponseParams');
+
+let ResponseParams: gax.protobuf.Type | null;
+try {
```

I'm a bit concerned with failing silently here due to a test failing. If the tests that are failing are only the tests for the samples I'd rather skip those tests in the CI until we figure out what to do

---

The trouble is that the error is for the entire suite so we would have to skip the entire CI check in order for this to be unblocked.

---

Is it only for the [samples](https://github.com/googleapis/nodejs-bigtable/tree/main/samples/test) test suite though? If it is I would still rather just skip those to be honest

---

If we really don't want to skip can we gate this behind an env variable that is set from the test?

---

Yes. Only for the samples suite.

We can change the npm script to do `cd samples/ && npm link ../ && npm install && npx cross-env METRICS_DEBUG=true npm test && cd ../` instead of `cd samples/ && npm link ../ && npm install && npm test && cd ../` and that might work. Do you want to do this?